test/parallel/test-promises-unhandled-rejections.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
```

Comment:
I have only reformated the first string arguments in each function, nothing more. GitHub diff just goes astray here, sorry.